tests/storage/datavolume.go

```diff
@@ -1262,7 +1262,7 @@ var _ = SIGDescribe("DataVolume Integration", func() {
 			Expect(virtClient.VirtualMachinePreference(virtualMachinePreference.Namespace).Delete(context.Background(), virtualMachinePreference.Name, metav1.DeleteOptions{})).To(Succeed())
 		})
 
-		It("should use PreferredStorageClassName when storage class not provided by VM", func() {
+		It("should use PreferredStorageClassName when storage class not provided by VM", decorators.Conformance, func() {
```

Comment:
can this be demoted to a unit test? It does not seem worthwhile to start a VM just for checking the order of preferences.

Reply:
I don't see a reason why not